suites/pacific/cephfs/tier-2_cephfs_test-nfs.yaml

```diff
@@ -299,3 +299,9 @@ tests:
       module: nfs.nfs_export_config.py
       name: "cephfs nfs export creation using config file"
       polarion-id: "CEPH-83574008"
+  - test:
+      abort-on-fail: true
```

Comment:
Do we need to abort further tests on failure of this particular test? since cleanup is handled in finally block can we have the 'abort_on_fail' as false? Open to discussion.

Reply:
No, This was consequence of copy and paste error from above test case.
Changed all of them. Thanks Neha